1-src/api/auth/auth-utilities.mts

```diff
@@ -6,21 +6,45 @@ import { query, queryAll, queryTest, TestResult } from "../../services/database/
 import { DB_USER } from "../../services/database/database-types.mjs";
 import { RoleEnum } from "../profile/profile-types.mjs";
 import { formatProfile } from "../profile/profile-utilities.mjs";
+import JWT_PKG, { JwtPayload } from "jsonwebtoken";
+import { createHash } from 'node:crypto'
+import dotenv from 'dotenv';
+dotenv.config(); 
+
+const {sign, verify} = JWT_PKG;
+
+/********************
+   Create secret key
+********************/
+var APP_SECRET_KEY;
+
+export const generateSecretKey = () => {
+   const time = new Date().getTime();
+   APP_SECRET_KEY = createHash("sha256").update(time + process.env.SECRET_KEY).digest("base64");
+}
 
```

Comment:
Go ahead and initialize generateSecretKey() here after the declaration.  At the bottom of the file is too far away to remember the connection.